Project/Assets/Editor/Annotations/AnnotationInteractionEditor.cs

```diff
@@ -0,0 +1,147 @@
+﻿using UnityEngine;
+using UnityEditor;
+using System.IO;
+
+
+[CustomEditor(typeof(AnnotationInteraction))]
+public class AnnotationInteractionEditor : Editor {
+
+    bool showRichText = true;
+    AnnotationInteraction annotation;
+
+
+    public override void OnInspectorGUI()
+    {
+        annotation = (AnnotationInteraction)target;
+
+        string[] options = new string[] { "Import Text File", "Write in Inspector" };
+        annotation.selected = EditorGUILayout.Popup("Import Method", annotation.selected, options);
+
```

Comment:
The ability to select via dropdown which kind of import you want is a top notch way to go about this.

_Note (that we don't need to solve now):_ selecting 'import text file', selecting a file, then switching to 'write in inspector', editing the text, and then switching back can lead to kinda strange behavior. The text file is still considered linked (appears in the GUI), but the text inside the box is different from the text file (because it is still sourced from the user inputed text).